elasticache-redis/tests/unit.tftest.hcl

```diff
@@ -191,6 +191,21 @@ run "aws_ssm_parameter_unit_test" {
   }
 }
 
+run "aws_ssm_parameter_ssl_unit_test" {
+  command = plan
+
+  ### Test aws_ssm_parameter ssl resource ###
+  assert {
+    condition     = aws_ssm_parameter.endpoint_ssl.name == "/copilot/redis-test-application/redis-test-environment/secrets/REDIS_TEST_NAME_URL"
+    error_message = "Invalid config for aws_ssm_parameter ssl name"
+  }
+
+  assert {
+    condition     = aws_ssm_parameter.endpoint_ssl.type == "SecureString"
+    error_message = "Invalid config for aws_ssm_parameter type"
+  }
```

Comment:
Can we assert on the value and the tags too?

Reply:
- Tests for tags have been added for both ssm parameters.
- the ssm parameter value can only be retrieved by running a terraform apply test.

Reply:
e2e tests for
- aws_ssm_parameter.endpoint_ssl.value &
- aws_ssm_parameter.endpoint.value
are proving to be flakey, passinhg/failing intermittently.

I have discussed with @antroy-madetech and due to the length of time it takes to run the e2e test decided to add the comment to the unit tests about the value attribute only being able to be tested when running terraform apply.